RockPaperScissors/RockPaperScissors/Enum/MukchippaResult.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  MukchippaResult.swift
+//  RockPaperScissors
+//
+//  Created by nayeon  on 2023/11/29.
+//
+
+import Foundation
+
+enum MukchippaResult: Int {
+    case draw = 0
+    case computerWin = 1
+    case computerLose = 2
```

Comment:
```suggestion
    case computer = 1
    case user = 2
```
의 형태로 구현해보는 건 어떨까요??
현재 currentTurn의 타입이 MukchippaResult이라 현재 누구의 턴인지 명확하게 보여주면 어떨까 싶어 제안드려요!
컴퓨터 승/패만을 나타내는 것 같아서 묵찌빠와 관련된 세부로직들을 볼때 한번에 이해하기 조금 어려웠던 것 같아요!
이렇게 구현해보시고 어떻게 느끼셨는지 알려주시면 더 좋을 것 같아요!

Reply:
처음에는 RockPaperScissorsResult을 활용해서 (컴퓨터와 사용자가 이기고 지는게 정 반대고 draw는 똑같으니까요) 사용해볼려고 했는데 그렇다면 네이밍이 직관적이지 않아서 열거형을 만든부분이라 똑같이 맞출려고 했던것같아요! 이 부분이 훨씬 직관적인것같아요!